src/store/store.ts

```diff
@@ -0,0 +1,1448 @@
+// Pinia Store
+import {
+  AccountTypes,
+  ActionTypes,
+  ApprovalTypes,
+  CoopTypes,
+  CorrectionErrorTypes,
+  FilingNames,
+  FilingTypes,
+  PartyTypes,
+  RelationshipTypes,
+  RestorationTypes,
+  RoleTypes
+} from '@/enums/'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module/'
+import {
+  AddressesIF,
+  BusinessInformationIF,
+  CertifyIF,
+  CorrectionInformationIF,
+  EffectiveDateTimeIF,
+  EntitySnapshotIF,
+  FeesIF,
+  FilingDataIF,
+  FlagsCompanyInfoIF,
+  FlagsReviewCertifyIF,
+  NameRequestIF,
+  NameTranslationIF,
+  OrgPersonIF,
+  ResolutionsIF,
+  ResourceIF,
+  RestorationStateIF,
+  ShareClassIF,
+  StateIF,
+  StateFilingRestorationIF,
+  ValidationFlagsIF,
+  ActionKvIF,
+  CourtOrderIF } from '@/interfaces/'
+import {
+  CompletingPartyIF,
+  ContactPointIF,
+  NaicsIF,
+  SpecialResolutionIF,
+  StaffPaymentIF
+} from '@bcrs-shared-components/interfaces/'
+import { IsSame } from '@/utils/'
+import DateUtilities from '@/services/date-utilities'
+
+import { defineStore } from 'pinia'
+import { resourceModel, stateModel } from './state'
+import { LegalServices } from '@/services'
+
+export const useStore = defineStore('store', {
+  // convert to a function
+  state: (): StateIF => ({
+    stateModel: stateModel,
+    resourceModel: resourceModel
+  }),
+  getters: {
```

Comment:
Given that getters and actions are separate objects anyway, I'm leaning towards separate files/imports for them. I see no advantage of having everything in this one file.

Reply:
We can move them after this PR - I'd rather keep them together and have smaller stores - but it is possible:

https://github.com/vuejs/pinia/issues/802#issuecomment-1018780409

Reply:
Thanks. I see you got the last word though 😆